content/en/providers/operations-manuals/howto21_adding_service_endpoint_into_gocdb/_index.md

```diff
@@ -0,0 +1,175 @@
+---
+title: HOWTO21
+weight: 135
+type: "docs"
+description: "How to add URL service endpoint information into GOC-DB"
+---
+
+For monitoring purposes, each service endpoints registered into GOC-DB
+with the flags *production* and *monitored* should include the endpoint
+URL information in order to be contacted by the relative nagios probe.
+
+**Important: currently these steps are required only for the sites under
+certification. For sites in production, EGI Monitoring is retrieving
+from the Top-BDII these information**
+
+The information needed for service type are:
+
+  - CREAM-CE: the queue URL where the ops VO is enabled. This is the
+    value of the attribute GlueCEUniqueID published in the BDII (e.g.
+    *cream.egi.eu:8443/cream-pbs-ops*).
+  - SRM: the value of the attribute GlueServiceEndpoint published in the
+    BDII (e.g. *httpg://se.egi.eu:8444/srm/managerv2*)
+  - Other service types: the value of the attribute GlueServiceEndpoint
+    published in the BDII
+
```

Comment:
Missing the cloud service types info

Reply:
Added what I found in https://docs.egi.eu/providers/cloud-compute/registration/.